tests/parameters/test_categorical.py

```diff
@@ -11,16 +11,16 @@
 from parameterspace.parameters.base import BaseParameter
 from parameterspace.parameters.categorical import CategoricalParameter
 
-from .util import check_sampling, check_value_numvalue_conversion, check_values
+from .util import check_value_numvalue_conversion, check_values
 
 
 def test_categorical_parameter():
     values = [4, 3, 2, 1, 0]
 
     p = CategoricalParameter("foo", values, prior=[1] * len(values))
 
-    assert p.is_continuous == False
-    assert p.is_ordered == False
+    assert p.is_continuous is False
+    assert p.is_ordered is False
```

Comment:
Why `assert p.is_ordered is False` instead of `assert not p.is_ordered`?

Reply:
Just because I was lacy and it was easier to find & replace. :see_no_evil: 
I agree, your proposal is better. \*booting-regex-skills\* :mechanical_arm:

Reply:
Done